components/Hero/Hero.jsx

```diff
@@ -1,20 +1,30 @@
 import styles from '@/styles/Hero.module.css'
 import { Image } from '@/components'
+import { Sanitize } from '@/components'
 
 const Hero = (props) => {
     const { title, description, image } = props
+
     return (
         <div className={styles.container}>
             <div className={styles.hero_container}>
                 <div className={styles.welcome_box}>
-                    <h3 className={styles.welcome_text}>{title}</h3>
+                    <h3 className={styles.welcome_text}>
+                        <Sanitize input={title} />
+                    </h3>
                 </div>
                 <div className='body1'>
-                    <p className={styles.intro_text}>{description}</p>
+                    <p className={styles.intro_text}>
+                        <Sanitize input={description} />
+                    </p>
                 </div>
             </div>
             <div className={styles.crop_img_banner}>
-                <Image className={styles.img_banner} alt={image.alt} {...image} />
+                <Sanitize input={image.src} />
```

Comment:
you don't need to use on all the props. 
just on the content ones :)

Reply:
The Image component needs all props. @eliciusc, you can sanitize inside the Image component. Just alt and src.

Reply:
sanitizing on Image component